vital/api/link.py

```diff
@@ -76,3 +76,25 @@ def oauth_provider(
             f"/link/provider/oauth/{provider}",
             headers={"LinkToken": link_token},
         )
+
+    def demo_provider(
```

Comment:
why not conect_demo_provider?

Reply:
I just used the same style of the ones above - `oauth_provider`, `email_provider`, etc.